internal/usage/usage.go

```diff
@@ -453,7 +453,7 @@ dbName and collection are required only for built-in roles.`
 	EnableWatchDefault                        = "Flag that indicates whether to watch the command until it completes its execution or the watch times out."
 	WatchTimeout                              = "Time in seconds until a watch times out. After a watch times out, the CLI no longer watches the command."
 	CompactResponse                           = "Flag that enables the compact array response structure for a json output. The --compact option returns array objects as top-level responses and allows backward compatibility for scripts based on previous CLI versions. Omitting the --compact option for a json output returns array objects within a 'results' sub-array. You must specify --output json to use this option."
-	DataFederationFile                        = "Path to an optional JSON configuration file that defines data federation settings."
+	DataFederationFile                        = "Path to an optional JSON configuration file that defines data federation settings. . To learn more about configuration files for the Atlas CLI, see https://www.mongodb.com/docs/atlas/cli/stable/reference/json/data-federation-config-file/."
```

Comment:
flyby

Reply:
+1 for a dochub

Reply:
https://dochub.mongodb.org/core/atlas-cli-json-data-federation-config